themes/default/content/blog/introducing-azure-native-v2/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Introducing Azure Native 2.0"
+
+date: 2023-06-13T08:00:00-04:00
+
+meta_desc: "Introducing Pulumi Azure Native Provider 2.0: Streamlined, Expanded, and More Powerful than Ever!"
+
+meta_image: meta.png
+
+authors:
+    - monica-rodriguez
+
+tags:
+    - providers
+    - azure
+
+---
+
+{{% notes type="warning" %}}
+The Pulumi Azure Native Provider 2.0 is currently available in beta.
+{{% /notes %}}
+
+We are thrilled to announce the highly anticipated beta release of the Pulumi Azure Native Provider 2.0, a significant upgrade to our infrastructure as code (IaC) solution for Microsoft Azure. This release brings a host of exciting features and improvements that will enhance your experience with managing Azure resources and empower you to build robust and scalable cloud infrastructure more efficiently.
+
+<!--more-->
+
+At Pulumi, we understand the importance of keeping up with the ever-evolving cloud landscape. The Pulumi Azure Native Provider 2.0 represents our commitment to providing you with the best tools to harness the full power of Microsoft Azure while providing you with same-day access to the entire [Azure API surface](https://docs.microsoft.com/en-us/rest/api/azure/). Let's dive into some of the key enhancements in this release:
+
+## Reduced SDK Size for Faster Deployments
+
+We've listened to your feedback and have made significant optimizations to the Azure Native Provider SDKs. With careful fine-tuning and restructuring, we have achieved a remarkable decrease in SDK size of up to 60% <!-- TODO: Get latest size updates on alpha.3 once released -->. This improvement translates into faster deployment times, reduced resource consumption, and an overall smoother experience when working with the provider.
+
+So, how did we accomplish this? For starters, we've removed all deprecated ARM API resources <!-- TODO: Link to list of deprecated resources --> up to the 2.0 release date. The bigger impact was made by identifying and removing all forward-compatible resource versions. That is to say we've exluded any redundant versions and kept the latest version until breaking changes. Where that version is not the current latest and new default, the explicit versions are still available.
+
+![version_squeeze_diagram](./azurev2-diagrams.png)
+
+## Refreshed Default Resource Versions
+
+In the latest release, we have refreshed the default resource versions to align with the latest ARM API versions from Microsoft. With our continued committment to same-day access to Azure API surface, you always have access to the most up-to-date features and capabilities offered by Azure resources. By leveraging the latest resource versions, you can take advantage of the latest and most desired features directly from Azure while building your infrastructure with Pulumi.
```

Comment:
Perhaps we need a paragraph before this explaining what "default versions" are? Something like this maybe.

> When Azure releases new features they often create a new version of a service. These versions named using dates such as "2021-01-02" or "2022-05-08-preview". There's many versions of each resource in the SDK, so we pick a default version of each resource 

I've also found this page hidden in the registry docs which we should also reference: https://www.pulumi.com/registry/packages/azure-native/version-guide/

Interestingly that page uses the termonology of "Top-Level Modules" and "Module Per Version". Perhaps we should adjust our language to match with that?